gopherage/OWNERS

```diff
@@ -2,13 +2,12 @@
 
 reviewers:
 - spiffxp
-- BenTheElder
 - MushuEE
 approvers:
 - spiffxp
-- BenTheElder
 emeritus_approvers:
 - amwat
+- BenTheElder
```

Comment:
not focusing on this and other engprod tools I'm not actively interested in, too many things to do. this and tools below have other listed owners, plus repo root OWNERS.